catalog/dags/common/slack.py

```diff
@@ -396,15 +396,15 @@ def on_failure_callback(context: dict) -> None:
             log.info("Forgoing Slack alert due to upstream failures")
             return
         exception_message = f"""
-*Exception*: {exception}
-*Exception Type*: `{exception.__class__.__module__}.{exception.__class__.__name__}`
+*Exception Type*: `{exception.__class__.__module__}.{exception.__class__.__name__}
+*Exception*: {exception}`
```

Comment:
Looks like the trailing `` ` `` got lost in the mix here.

```suggestion
*Exception Type*: `{exception.__class__.__module__}.{exception.__class__.__name__}`
*Exception*: {exception}
```